### PR TITLE
Add branch strategy to README

### PR DESCRIPTION
**Pull request by goneall**
This implements the branching strategy agreed to on the [5 November 2024 tech call](https://github.com/spdx/meetings/blob/main/tech/2024/2024-11-05.md#branch-policy).

### CODE REVIEW

**Review by bact — CHANGES_REQUESTED** [review]
Thank you. I have made few minor suggestions inline.

**Comment by goneall** [comment]
@bact - Thanks for the suggestions - all have been accepted

**Review by bact — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -60,6 +60,17 @@ This repository consists of these files and directories (partial):
 - `mkdocs.yml` - MkDocs recipe for the spec documentation generation. The
   inclusion of model files and the order of chapters are defined here.
 
+## Branch structre
+
+The SPDX spec repo follows the [Gitflow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow) workflow with the addition of support branches.
```
Comment (zvr) [inline README.md:65]: Shall we link to something less critical of Gitflow?
For example https://gist.github.com/HeratPatel/271b5d2304de2e2cd1823b9b62bf43e0 (which also incudes support branches)

**Review by zvr — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (goneall) [reply README.md:65]: Yes - I was looking for a better link - so thanks for providing a good alternative.   I was hoping to find something in Wikipedia - but the only references are to the URL I used above.

I'll update the PR.

Comment (goneall) [reply README.md]: PR Updated

**Comment by goneall** [comment]
Since the suggestion from @zvr were accepted and we should have this published with the branches renamed, I'm going to go ahead and merge.

If there are further suggestions, we can create a separate PR.

**Comment by goneall** [comment]
@bact - Thanks for catching this - the master branch has now been renamed to main

**Pull request merged**